docs/eventing/README.md

```diff
@@ -72,15 +72,16 @@ To learn how to use the registry, see the
 Knative Eventing also defines an event forwarding and persistence layer, called
 a
 [**Channel**](https://github.com/knative/eventing/blob/master/pkg/apis/eventing/v1alpha1/channel_types.go#L36).
```

Comment:
this link does not exist anymore, and since each Channel is now a separate CRD, not sure if we should instead point to the Channel Spec.
@Harwayne 
@matzew

Reply:
Good catch! I will checking for other links with pattern `pkg/apis/eventing`

https://github.com/knative/eventing/blob/master/pkg/apis/messaging/v1alpha1/channel_types.go#L57